File: 2018-summer/week1/H.java
/*************************************************************************
	> File Name: H.java
	> Author: Li Meng
	> Mail: ldx19980108@gmail.com
	> Created Time: 2018年07月24日 星期二 11时39分23秒
 ************************************************************************/

import java.util.Scanner;

//OJ提交类名为Main
//public class Main {
public class H {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int  n = input.nextInt();
        input.nextLine();
        for(int i=0; i<n; i++){
            String str = input.nextLine();
            if(str.matches("^[a-zA-Z_][a-zA-Z0-9_]*$")){
                System.out.println("yes");
            }else {
                System.out.println("no");
            }
        }
    }
}

File: 2020-SE/2/H.cpp
/*************************************************************************
	> File Name: H.cpp
	> Author:Li Meng
	> Mail:ldx19980108@gmail.com
	> Created Time: 一  9/28 12:21:13 2020
 ************************************************************************/

#include<iostream>
using namespace std;

int main(void)
{
    int i, j, k;
    for(i = 0; i <= 33; i++){
        for(j = 0; j <= 50; j++){
            if((9*i + 6*j + 100 - i -j) == 300){
                cout << i << " " << j << " " << 100 - i - j << endl;
            }
        }
    }
    return 0;
}
